src/helpers/optionalStacking/common.ts

```diff
@@ -28,24 +37,111 @@ export function getViewByTwoAttributes(
  *      => ignore computeRatio and use stackMeasuresToPercent
  *
  * stackMeasures is applied only when there are [2 measures and more]
- * stackMeasuresToPercent is applied only when there are [1 measure + 1 stackBy] or [2 measures and up]
+ * stackMeasuresToPercent is applied only when there are [1 measure + 1 stackBy] or [2 measures and more]
  */
 export function getSanitizedStackingConfig(
```

Comment:
previous name getSanitizedStackingConfig reflected the fact, that it returns new ChartConfig. But new implementation returns config + new buckets, so its name is no longer valid. What about "getSanitizedBucketsAndStackingConfig"? I know its long :(

Reply:
updated